src/frontends/jax/src/op/iota.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/jax/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/range.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace jax {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_iota(const NodeContext& context) {
+    num_inputs_check(context, 2, 2);
+    auto dtype = context.const_named_param<ov::element::Type>("dtype");
+    auto size = context.const_named_param<int64_t>("size");
```

Comment:
are we sure that `size` is attribute of `iota` operation and it can not be non-constant input?
Can you please try to experiment with it and create Jax graph with size input to iota operation?

Reply:
sure thing!, I will experiment it on my end & will update soon.
thanks